build-tools/rpm/macros.yast

```diff
@@ -110,6 +111,7 @@ fi \
         for f in $(%{_bindir}/find %{buildroot}/%{yast_desktopdir}/ -name '*.desktop') ; do \
             d=${f##*/} \
             %suse_update_desktop_file -d ycc_${d%.desktop} ${d%.desktop} \
+            %{yast_ydatadir}/devtools/bin/y2metainfo -f ${f} -l %{license} -o %{yast_metainfodir}/ \
```

Comment:
Time to write a filelist generator >:D

Joking aside, it might be better to introduce it in modules manually, but I worry some of them would be missed without the reminder that they aren't building anymore :P 

Yup, missed the quotes around the license